### PR TITLE
Serve all campaigns from Phoenix.

### PR DESCRIPTION
**Pull request by DFurnes**
This pull request removes the `ashes_campaigns` dictionary, so all legacy campaigns will now be served by Phoenix. Since we will no longer be serving any campaigns from Ashes, I'll [unschedule](https://app.ghostinspector.com/tests/5accf87d0c820a0234ba5b00) the Ashes campaign experience monitoring test (on "Baby It's Cold Inside") before applying this.

References [#163062247](https://www.pivotaltracker.com/story/show/163062247).

### CODE REVIEW

File: dosomething/fastly-frontend/ashes_init.vcl [hunk 1]
```diff
@@ -1,295 +0,0 @@
...
-  "whos-next": "true",
-  "wildlife-cards": "true",
-  "world-recycle-week-close-loop-0": "true",
-  "would-you-rather": "true",
-  "yo-teach": "true",
-  "youth-wave-week-action": "true",
-  "zombie-blood-drive": "true",
-
-  # international campaigns on /campaigns prefix:
-  "sua-mochila-pesada-pode-te-causar-muitos-problemas": "true",
```
[the remark this answers is not in the document]
Comment (DFurnes) [reply dosomething/fastly-frontend/ashes_init.vcl]: And depending on the answer to this, do we want to stop `/voluntario` routing to Ashes as well at some point? What's the deprecation plan for these old campaigns?

[the remark this answers is not in the document]
Comment (mshmsh5000) [reply dosomething/fastly-frontend/ashes_init.vcl]: I don't see either of these on [their campaigns listing](https://www.dosomething.org/mx/campaigns), so I think we're okay to kill them.

File: dosomething/fastly-frontend/ashes_recv.vcl [hunk 2]
```diff
@@ -26,12 +26,7 @@ else if (req.url.path ~ "(?i)\/((us|mx|br)\/)?(fact|sobre|volunteer|voluntario|r
...
   set req.http.X-Fastly-Backend = "ashes";
 }
-else if (req.url.path ~ "(?i)\/((us|mx|br)\/)?campaigns/([A-Za-z0-9_\-]+)" &&
-    table.lookup(ashes_campaigns, std.tolower(re.group.3))) {
-  # See if a given campaign should be served by Ashes:
-  set req.http.X-Fastly-Backend = "ashes";
-}
 else if (req.url.path ~ "(?i)^\/robots\.txt") {
-  # Finally, serve robots.txt rom Ashes on production:
+  # Finally, serve robots.txt from Ashes on production:
```
Comment (weerd) [inline dosomething/fastly-frontend/ashes_recv.vcl]: `rom` 🤖

image

Comment (weerd) [reply dosomething/fastly-frontend/ashes_recv.vcl]: 😂

**Comment by DFurnes** [comment]
Rebased against `master` (above), and gonna pull the lever! 🚀

**Comment by DFurnes** [comment]
Applied & confirmed all campaigns are now loading on Phoenix! 🎉

**Comment by DFurnes** [comment]
Finally, unscheduled the old Ashes campaign flow monitor (linked above).

**Pull request merged**